lib/glific/flags.ex

```diff
@@ -38,7 +35,9 @@ defmodule Glific.Flags do
   end
 
   defp out_of_office_check do
-    {:ok, now} = DateTime.now(@timezone)
+    timezone = Glific.Partners.organization().timezone
```

Comment:
@anshulrr can you make a alias for glific partners and use it in the complete file?